devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Install RPM packages
+  dnf:
+      name: ['git', 'vim', 'poetry', 'python3-pip', 'freeipa-server']
+      state: present
+
+- name: install python deps with poetry
+  shell: poetry install
+  become: yes
+  become_user: vagrant
+  args:
+    chdir: /vagrant/
+
+- name: install freeipa server
+  shell: ipa-server-install -a adminPassw0rd! --hostname=ipa.fas2ipa.test -r FAS2IPA.TEST -p adminPassw0rd! -n fas2ipa.test -U
+
+- name: get freeipa-fas
+  git:
+    repo: https://github.com/fedora-infra/freeipa-fas.git
+    dest: /home/vagrant/freeipa-fas
+
+- name: install freeipa-fas
+  shell: ./install.sh
+  args:
+    chdir: /home/vagrant/freeipa-fas/
+
+- name: ipa-backup for restoring
+  shell: ipa-backup
+
+- name: Install the .bashrc
+  copy:
+      src: .bashrc
+      dest: /home/vagrant/.bashrc
+      mode: 0644
+      owner: vagrant
+      group: vagrant
+
+- name: Install the config.toml
+  copy:
+      src: /vagrant/config.toml.example
```

Comment:
That's actually not necessary, the script will load the `config.toml.example` file first and the `config.toml` file second, so it changes nothing if they are identical. On the other hand having this copy step here will overwrite the `config.toml` file if there was one, which is not necessarily what we want.